src/controllers/userAnswerController.ts

```diff
@@ -0,0 +1,98 @@
+import { UserAnswer, User } from '../models/';
+import { Request, Response, NextFunction } from 'express';
+import { NotFoundError, BadRequestError, UnauthorizedError } from '../errors/httpErrors';
+import { UniqueConstraintError } from 'sequelize';
+import bcrypt from 'bcryptjs';
+
+async function retrieveUserAnswer(req: Request, res: Response, next: NextFunction) {
+  try {
+    const userAnswer = await UserAnswer.findByPk(req.params.id);
+    if (userAnswer === null) {
+      throw new NotFoundError('Security Question cannot be found');
+    }
+    req.userAnswer = userAnswer;
+    next();
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function indexUserAnswer(req: Request, res: Response, next: NextFunction) {
+  try {
+    const userAnswers = await UserAnswer.findAll();
+    res.status(200).json(userAnswers);
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function showUserAnswer(req: Request, res: Response, next: NextFunction) {
+  try {
+    res.status(200).json(req.userAnswer);
+  } catch (err) {
+    next(err);
+  }
+}
+
+async function createUserAnswer(req: Request, res: Response, next: NextFunction) {
+  try {
+    if (req.body.cotent) req.body.content = req.body.content.trim(); // Remove whitespace before hashing
+
+    const userAnswer = await UserAnswer.create(req.body);
+    res.status(201).json(userAnswer);
+  } catch (err) {
+    if (err instanceof UniqueConstraintError)
+      next(new BadRequestError('The User already has already used this question!'));
+    next(err);
+  }
+}
+
+async function updateUserAnswer(req: Request, res: Response, next: NextFunction) {
```

Comment:
We should remove this endpoint. A user could be logged into a public computer and forgets to logout. Somebody else could walk by and change the security question answer to reset the user's password.